docs/developers/1_overview.md

```diff
@@ -0,0 +1,24 @@
+# 💁 Overview
+Take a quick peak of how ReVanced Compose work.
+
+## ⚙️ Technology
+We use [Jetpack Compose](https://developer.android.com/jetpack/compose) to 
+build beautiful and performant user interfaces using declarative programming. 
+It provides a unified and efficient way of building UI that is well-integrated with the Android framework. 
+
+## 📂 Structure
+We structure our code to use [MVVM architecture](https://developer.android.com/topic/libraries/architecture/viewmodel) for easier maintenance of the code. 
```

Comment:
This can be put into the same section as a technology stack section. See how other Jetpack Compose apps do this.